src/Tribe/RSVP.php

```diff
@@ -1650,32 +1650,6 @@ public function get_ticket_reports_link( $unused_ticket_id, $event_id_deprecated
 	}
 
 	/**
-	 * Get an array of ticket IDs associated with an event
-	 *
-	 * @param  int $event_id   ID of the event in question
-	 * @return array           array of ticket IDs
-	 */
-	public function get_tickets_ids( $event_id ) {
```

Comment:
Deprecate?

Reply:
Not deprecated; I moved it in to the Abstract; 
https://github.com/moderntribe/event-tickets/pull/530/files#diff-871d327ba626e7a6e90c6c90572a8169R281